Sources/YapDB.swift

```diff
@@ -124,7 +130,7 @@ extension YapDB {
     A wrapper around YapDatabaseView. It can be constructed with a name, which is
     the name the extension is registered under, a grouping enum type and a sorting enum type.
     */
-    public class View: BaseView, YapDatabaseViewProducer, YapDatabaseExtensionRegistrar {
+    open class View: BaseView, YapDatabaseViewProducer, YapDatabaseExtensionRegistrar {
```

Comment:
I think this should be `open` as people can use this class to wrap Yap extensions not yet wrapped by this pod.